src/assisted_test_infra/test_infra/helper_classes/config/day2_cluster_config.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC
+from dataclasses import dataclass
+
+from assisted_test_infra.test_infra.helper_classes.config.cluster_config import BaseClusterConfig
+
+
+@dataclass
+class BaseDay2ClusterConfig(BaseClusterConfig, ABC):
+    day1_cluster_id: str = None
+    day1_cluster_name: str = None
+    num_day2_workers: int = None
```

Comment:
Change to `day2_workers_count` or `workers_count_day2` to be consistent with `ClusterConfig.workers_count`

Reply:
done